File: Data Structures & Algorithms/concatenation-of-array/submission-0.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int l= nums.length;
        int[] ans = new int[2*l];
        for(int i=0;i<l*2;i++)
        {
            ans[i] = nums[i%l];
        }
        
        return ans;
    }
}

File: Data Structures & Algorithms/majority-element-ii/submission-0.py
class Solution:
    def majorityElement(self, nums: List[int]) -> List[int]:
        l=len(nums)
        majl=l//3
        arr={}
        ans=[]
        for i in nums:
            arr[i]=arr.get(i,0)+1
            if arr.get(i)>majl and i not in ans:
                ans.append(i)
        return ans

        









        